.github/workflows/build_test.yml

```diff
@@ -0,0 +1,112 @@
+name: Standard Build and Test
+
+on:
+  # allows us to run workflows manually
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - develop
+      - master
+  push:
+    branches:
+      - develop
+      - master
+
+jobs:
+  Housekeeping:
+    runs-on: ubuntu-latest
+    container:
+      image: svalinn/dagmc-ci-ubuntu-18.04-housekeeping:latest
+    steps:
+      - name: Install latest git
+        run: |
+          apt-get install -y software-properties-common
+          add-apt-repository ppa:git-core/ppa
+          apt update
+          apt install -y git
+
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Setup environment 
+        run: |
+          echo "REPO_SLUG=${GITHUB_REPOSITORY}" >> $GITHUB_ENV
+          echo "PULL_REQUEST=$(echo  $GITHUB_REF | cut -d"/" -f3)" >> $GITHUB_ENV
+          mkdir /root/build_dir
+          ln -s $GITHUB_WORKSPACE /root/build_dir/DAGMC
+     
+      - name: Housekeeping
+        run: |
+          cd $GITHUB_WORKSPACE
+          CI/circleci/housekeeping.sh
+
+
+  BuildTest:
+    needs: Housekeeping
+    runs-on: ubuntu-latest
```

Comment:
(b/c I don't know:) do we need this `runs-on` if we specify the images below?

Reply:
yes that the host on which we run the docker image